circuit/merkle.circom

```diff
@@ -37,8 +37,8 @@ template RootFromMerklePath( maxDepth ) {
   signal aux[ maxDepth+1 ];
   aux[0] <== leaf;
 
-  // compute which prefixes (in big-endian) of the index is
-  // the same as the corresponding prefix of the last index
+  // compute which binary postfixes of the index is the same as the
+  // corresponding postfix of the last index
```

Comment:
While this technically correct, I would prefer some wording which better explains what happens here.

This is used for determining the bit in the key which should be 1 if the node is both the last one in the row and odd.

Also, since humans write numbers with the most significant digits first (including binary numbers), I think it's just more intuitive for the brain to think about the digits that way, even though in code it's better to do the opposite.

Reply:
Makes sense. I've updated the comment to better capture the intent, hopefully.